test/dlc_tests/sanity/test_telemetry.py

```diff
@@ -130,6 +131,38 @@ def test_pytorch_inference_job_type_env_var(pytorch_inference):
         container_name_prefix=container_name_prefix
     )
 
+def _run_s3_query_bucket_success(image_uri, ec2_client, ec2_instance, ec2_connection):
+    ec2_instance_id, _ = ec2_instance
+    account_id = test_utils.get_account_id_from_image_uri(image_uri)
+    image_region = test_utils.get_region_from_image_uri(image_uri)
+    repo_name, image_tag = test_utils.get_repository_and_tag_from_image_uri(image_uri)
+    framework, framework_version = test_utils.get_framework_and_version_from_tag(image_uri)
+    job_type = test_utils.get_job_type_from_image(image_uri)
+    processor = test_utils.get_processor_from_image_uri(image_uri)
+    container_type = test_utils.get_job_type_from_image(image_uri)
+    container_name = f"{repo_name}-telemetry_s3_query_success-ec2"
+
+    docker_cmd = "nvidia-docker" if processor == "gpu" else "docker"
+    test_utils.login_to_ecr_registry(ec2_connection, account_id, image_region)
+    ## For big images like trcomp, the ec2_connection.run command stops listening and the code hangs here.
+    ## Hence, avoiding the use of -q to let the connection remain active.
+    ec2_connection.run(f"{docker_cmd} pull {image_uri}")
+
+    actual_output = import_framework(image_uri, container_name, docker_cmd, framework, job_type, ec2_connection, test_mode = 1)
+
+    py_version = ec2_connection.run(
```

Comment:
why do you need to get python version from container instead of image uri? i believe we already have a test that tests whether python version in the tag is equivalent to python version in the container.

Reply:
that python version format is different than what we send for telemetry, for example telemetry needs 3.8.13, not py38 which comes from image uri